orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -103,13 +105,14 @@ public long getBackoffPeriod() {
 
   @Override
   public long getTimeout() {
-    return TimeUnit.MINUTES.toMillis(5);
+    return TimeUnit.MINUTES.toMillis(timeout == null ? 5 : timeout);
   }
 
   @Nonnull
   @Override
   public TaskResult execute(@Nonnull StageExecution stage) {
     StageData stageData = stage.mapTo(StageData.class);
+    timeout = stageData.timeout;
```

Comment:
can be removed after you override `getDynamicTimeout`

Reply:
addressed